kwanj-k/storemanager-API
# #161415254 Add message assertions and tests and fix admin required decorator

### What does this PR do?
Adds message assertions in tests and fixes the admin decorator

#### Description of Task to be Completed ?
-- Check for message response in tests
-- Fix admin required decorator so that only admins can add products

### How should this be manually tested?
-- * After cloning the repo,git pull the branch bg-tests-message-assert-#161415254
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:

  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

 All the tests should pass even with the new assertions

### What are the relevant Pivotal Tracker stories?
 #161415254
